### PR TITLE
Accounts Payable Frontend Implementation

### LINKED ISSUE #4: As a user, I want to track all financials so that I can be aware of the states of my accounts
User Points: 10

Risk: High. Although the implementation of this user story will work, if it doesn't, the impact of it will be high and the likelihood of it breaking is likely. 

Priority: High. It is a core feature that will affect which other features a user have access to.

Description of the requirement:
The system must allow following all and any cost: Accounts Payable, Accounts Receivable, and Cost/Charge Number.

Acceptance Test:

* User can track all financials

### PR DESCRIPTION
**Pull request by emilie-martin**
resolves #4 , alongside #103 
* added accounts payable page
* added accounts receivable page

Accounts Payable preview:
image

Accounts Receivable preview:
image

### CODE REVIEW

**Review by Manraj45 — CHANGES_REQUESTED** [review]
Remove space between imports and add comments

**Review by nasaku898 — APPROVED** [review]

File: bike-erp/backend/src/main/app.ts [hunk 1]
```diff
@@ -27,7 +26,6 @@ app.use(express.json());
 
 //Setup routes
 app.use("/", indexRouter);
-app.use("/account_receivable", accountReceivableRouter);
```
Comment (Manraj45) [inline bike-erp/backend/src/main/app.ts:30]: Why did you remove accoubt_receivable here?

**Review by Manraj45 — APPROVED** [review]
Great job!

File: bike-erp/backend/src/main/app.ts (hunk 1 above)
Comment (nasaku898) [reply bike-erp/backend/src/main/app.ts:30]: it was just sample route that sam made
